members/settings.py

```diff
@@ -75,12 +75,11 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'directory',
-        'USER': 'directory_user',
-        'PASSWORD': 'TAF_4each',
-        'HOST': 'localhost',
-        'PORT': '',
+        'ENGINE': 'django.db.backends.postgresql',
```

Comment:
@drewatk Were these changes made by docker? I think this is still correct, just curious.

Reply:
In the readme I linked the docker-compose guide I used